pkg/acsengine/const.go

```diff
@@ -78,9 +78,10 @@ const (
 	DCOSPublicAgent DCOSNodeType = "DCOSPublicAgent"
 )
 
-// KubeImages represents Docker images used for Kubernetes components based on Kubernetes version
+// KubeImages represents Docker images used for Kubernetes components based on Kubernetes version hint
+// version hint here is the major and minor part of the version. For instance, version 1.6.6 has version hint 1.6
 var KubeImages = map[string]map[string]string{
-	api.Kubernetes171: {
+	api.KubernetesVersionHint17: {
```

Comment:
Btw, i have split KubeImages into KubeImages in api package, and KubeConfigs in place.